src/app/common/navigation.service.ts

```diff
@@ -9,17 +9,19 @@ export class NavigationService {
     $.fn.setupVerticalNavigation ? $.fn.setupVerticalNavigation() : '';
   }
 
-  show() {
-    $.fn.setupVerticalNavigation
-      ? $.fn.setupVerticalNavigation().showMenu()
-      : '';
+  expand() {
+    const navElement = $('.nav-pf-vertical');
+    const bodyContentElement = $('.container-pf-nav-pf-vertical');
+
+    navElement.removeClass('collapsed');
+    bodyContentElement.removeClass('collapsed-nav');
```

Comment:
Just to double check, does the nav menu behave when you resize the browser in this case?  I thought I had tried this originally but the nav menu maintains some open/close state internally too which necessitated adding the `hideMenu()`/`showMenu()` functions.

Reply:
Ah let me check!

Reply:
Damn, you're right. Let me delete that PR and work something out...